### ISSUE TITLE
Confused with iteration beginning at 61801 & loading generator weights 🤔

### ISSUE THREAD

**Issue by RahulBhalley** [bojone/T-GANs#1]
```
g_train_model.load_weights('./g_train_model.weights')

for i in range(61801, total_iter):
```
In above code snippet from [t_sgan_sn_cifar10.py](https://github.com/bojone/T-GANs/blob/master/t_sgan_sn_cifar10.py) why is `g_train_model` `load`s the weight before training and the training initially begins from iteration step 61801?

**Comment by bojone** [comment]
because I break the training and test the model once. And then I continue to train it to see ultimate effect of it and I ignore it while sharing the code. I have repair it. Thank you very much.

**Comment by RahulBhalley** [comment]
Thanks for the reply! 🙂

Is this code written by the author of Turing GAN paper? If not, does this code produce the true results from the paper?

**Comment by bojone** [comment]
yeah. I am the author of Turing GANs and that is the original code of Turing GANs. It can reproduce the whole result in the paper. (Do not forget change the source code of Keras.)

**Comment by RahulBhalley** [comment]
WOW! 😲
I have one more question that if I re-write your Keras code in PyTorch then will I have to do any modifications to any internal code of PyTorch also like you suggested for Keras?

**Comment by bojone** [comment]
Change source code of keras is for spectral normalization. Because to change source code is the simplest way to implement spectral normalization in Keras. 

If in PyTorch, you can refer any spectral normalization implement of it, for example,  https://github.com/christiancosgrove/pytorch-spectral-normalization-gan. I am not familiar with PyTorch, so I can not give you further suggestions.

**Comment by RahulBhalley** [comment]
Thank you.
